.editorconfig

```diff
@@ -5,3 +5,9 @@ insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+
+[/Symfony/CS/Tests/Fixtures/**/*]
+end_of_line = ignore
+insert_final_newline = ignore
+indent_style = ignore
+trim_trailing_whitespace = ignore
```

Comment:
does `ignore` config value a valid one?
I cannot find it https://github.com/editorconfig/editorconfig/search?utf8=✓&q=ignore